lib/blockchain.go

```diff
@@ -63,24 +63,35 @@ const (
 	// don't store orphan headers and therefore any header that we do
 	// have in our node index will be known definitively to be valid or
 	// invalid one way or the other.
-	StatusHeaderValidated = 1 << iota
-	StatusHeaderValidateFailed
+	StatusHeaderValidated      = 1 << 0
+	StatusHeaderValidateFailed = 1 << 1
 
-	StatusBlockProcessed
-	StatusBlockStored
-	StatusBlockValidated
-	StatusBlockValidateFailed
+	StatusBlockProcessed      = 1 << 2
+	StatusBlockStored         = 1 << 3
+	StatusBlockValidated      = 1 << 4
+	StatusBlockValidateFailed = 1 << 5
+	StatusBlockCommitted      = 1 << 8
```

Comment:
Can you add descriptions for what each of these statuses means, and when each one is used? It'll make future maintainers' lives much easier